test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/GeneratorTests.cs

```diff
@@ -181,6 +181,69 @@ public void GenerateJsonDocumentTest_Relationship()
         Assert.AreEqual(expectedJsonContentAsString, generatedJsonString);
     }
 
+    [TestMethod]
+    public void GenerateJsonDocumentTest_PrereqFor_Relationship()
```

Comment:
Feels like we should test the direction both ways, since we have code to enforce it. Something like this:
[TestMethod]
[DataRow("source-id", "target-id")]
[DataRow("target-id", "source-id")]
public TestName(string sourceElementId, string targetElementId)
{
  var relatinshipInfo = new Relationship
  {
    SourceElementId = sourceElementId,
    TargetElementId = targetElementId,
...
```